account_invoice_report_service/views/report_invoice_delivery.xml

```diff
@@ -17,74 +17,81 @@
-                    <li>
-                        <span t-field="company.name" />
-                        -
-                        <t t-esc="company.country_id.vat_label or 'CIF'" />:
-                        <span t-field="company.vat" />
-                    </li>
-                    <li t-if="company.street or company.street2">
-                        <span
-                            t-esc="' '.join(filter(lambda x: x, [company.street,company.street2]))"
-                        />
-                        -
-                        <span
-                            t-esc="' '.join(filter(lambda x: x, [company.zip,company.city,company.state_id.name, company.country_id.name]))"
-                        />
-                    </li>
-                    <li>
-                        <t t-if="company.phone">
-                            Tel.
-                            <span t-field="company.phone" />
-                        </t>
-                        <span
-                            t-esc="' / '.join(filter(lambda x: x, [company.email, company.website]))"
-                        />
-                    </li>
-                </ul>
-                <div class="text-muted">
-                    Page
-                    <span class="page" />
-                    of
-                    <span class="topage" />
+            <div class="footer">
+                <div class="text-center" style="font-size: 10px;">
+                    <ul class="list-unstyled mt4 mb4">
+                        <li>
+                            <span t-field="company.name" />
+                            -
+                            <t t-esc="company.country_id.vat_label or 'CIF'" />:
+                            <span t-field="company.vat" />
+                        </li>
+                        <li t-if="company.street or company.street2">
+                            <span
+                                t-esc="' '.join(filter(lambda x: x, [company.street,company.street2]))"
+                            />
+                            -
+                            <span
+                                t-esc="' '.join(filter(lambda x: x, [company.zip,company.city,company.state_id.name, company.country_id.name]))"
+                            />
+                        </li>
+                        <li>
+                            <t t-if="company.phone">
+                                Tel.
+                                <span t-field="company.phone" />
+                            </t>
+                            <span
+                                t-esc="' / '.join(filter(lambda x: x, [company.email, company.website]))"
+                            />
+                        </li>
+                    </ul>
+                    <div class="text-muted">
+                        Page
+                        <span class="page" />
+                        of
+                        <span class="topage" />
+                    </div>
```

Comment:
Same here and in some other places, can you please check all the xml files and fix the spaces please?
Thanks!